boards/arm/samv7/same70-qmtech/configs/mcuboot-loader/defconfig

```diff
@@ -44,13 +47,12 @@ CONFIG_RAM_SIZE=262144
 CONFIG_RAM_START=0x20400000
 CONFIG_RAW_BINARY=y
 CONFIG_RR_INTERVAL=200
-CONFIG_SAME70QMTECH_FORMAT_MCUBOOT=y
-CONFIG_SAME70QMTECH_MCUBOOT_BOOTLOADER=y
+CONFIG_SAMV7_FORMAT_MCUBOOT=y
```

Comment:
`CONFIG_SAMV7_MCUBOOT_BOOTLOADER` -- is missing

Reply:
CONFIG_BOARDCTL_BOOT_IMAGE have same function.